src/Nakadi/Client/Stream.purs

```diff
@@ -136,6 +133,7 @@ handleRequest
 handleRequest { resultVar, buffer, bufsize } streamParams resStream commit eventHandler env = do
   callback <- splitAtNewline buffer bufsize  handle
   onData resStream callback
+  onEnd resStream (launchAff_ $ AVar.put StreamClosed resultVar)
```

Comment:
```suggestion
  onEnd resStream (launchAff_ $ AVar.put StreamClosed resultVar)
  onError resStream (\e -> do
    env.logWarn Nothing $ "Error in read stream " <> message e
    launchAff_ $ AVar.put StreamClosed resultVar
    )
```

Reply:
It's logged in retry part of `Client.purs`

Reply:
oops, sorry, you mean the error

Reply:
ah
so not?